src/Components/ServiceScene/Breakdowns/Patterns/PatternFieldLabelStats.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { LogLabelStatsModel, GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+//Components
+import { PatternFieldLabelStatsRow } from './PatternFieldLabelStatsRow';
+
+const STATS_ROW_LIMIT = 10;
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    logsStats: css`
```

Comment:
bit of a nit: but I think we're trying to stick to the css object styles in emotion instead of the plain css/string type: https://emotion.sh/docs/object-styles